### PR TITLE
Add default Boolean values in check reference

### LINKED ISSUE #2177: Checks reference: confirm Boolean defaults
Some Boolean attributes do not have a default value specified.

## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [ ] New content (guide wanted)
- [x] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

## Expected Behavior
For all Boolean attributes, list the default value.

## Current Behavior
Some Boolean attributes have a default value listed and some do not.

## Possible Solution
Specify the default value for all Boolean attributes.

## Affected Docs Pages
https://docs.sensu.io/sensu-go/latest/reference/checks/

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Specifies default Boolean values in check reference

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2177

## Review Instructions
I could not seem to produce a check that would result in a GET response that includes `discard_output`.

### CODE REVIEW

File: content/sensu-go/5.15/reference/checks.md [hunk 1]
```diff
@@ -586,6 +586,7 @@ description  | If check requests are published for the check.
 required     | false
 default      | `false`
 type         | Boolean
+default      | `false`
```
Comment (rgeniesse) [inline content/sensu-go/5.15/reference/checks.md:588]: I already see a default field here on line 587.

Also another odd thing I noticed between 5.15 and 5.18, the lines that type and default live on are reversed.

File: content/sensu-go/5.15/reference/checks.md [hunk 2]
```diff
@@ -586,6 +586,7 @@ description  | If check requests are published for the check.
 required     | false
 default      | `false`
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/reference/checks.md]:
```suggestion
```

File: content/sensu-go/5.15/reference/checks.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/5.15/reference/checks.md:588]: Thank you -- I should have removed that line.

Comment (hillaryfraley) [reply content/sensu-go/5.15/reference/checks.md:588]: I'm not sure I see what you mean by "between 5.15 and 5.18, the lines that type and default live on are reversed." I am probably overlooking it -- can you give an example?

Comment (rgeniesse) [reply content/sensu-go/5.15/reference/checks.md:588]: 5.15:



5.18:

Comment (hillaryfraley) [reply content/sensu-go/5.15/reference/checks.md:588]: Thank you! I do not know how I couldn't see that. I probably standardized the order when I copyedited everything.

**Review by rgeniesse — APPROVED** [review]

**Comment by hillaryfraley** [comment]
I moved the question about configuring a check that would result in a GET response that includes `discard_output` to https://github.com/sensu/sensu-docs/issues/2237.

**Pull request merged**